### ISSUE TITLE
Filtering incidents using `iter_all` by `service_ids` does not work

### ISSUE THREAD

**Issue by DeepSpace2** [PagerDuty/pdpyras#74]
```python
iter_all('incidents', params={'service_ids': [service_id]})
```
fails with `400` with the error

```python
'{"error":{"message":"Invalid Input Provided","code":2001,"errors":["Service ids must be a Array."]}}'
```

API V2 and `pdpyras 4.3.0`

**Comment by DeepSpace2** [comment]
Ok, got it to work by passing `params={'service_ids[]': [service_id]}` which is a bit weird. Can it be fixed?

**Comment by Deconstrained** [comment]
It is [required for set filters](https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTU2-filtering#set-filters) to add square brackets to the parameter name. Confusion over this came up once before and I thought it might be convenient to add them if the user forgets, but after a while with no new demand for it I closed the issue (#38). I've reopened it and will add this feature in a future update.
